contrib/cirrus/check_image.sh

```diff
@@ -36,4 +36,10 @@ do
         "$(systemctl list-unit-files --no-legend $REQ_UNIT)" = "$REQ_UNIT enabled" || let "RET+=1"
 done
 
+# Exits non-zero if any unit matching pattern is running
+UNIT_STATUS=$(systemctl is-active $EVIL_UNITS; echo $?)
+item_test "No interfearing background units are active:" \
+    "$UNIT_STATUS" -ne "0"
```

Comment:
Should there be a `|| let` here? (side note, there's no 'a' in 'interfering')

Reply:
OMG, sure did forget that. Thanks. Gosh so embarrassing.  At least I'm well use to it by now :smile:

Reply:
Are you aware that you could have `item_test` do the updating?

Reply:
Certainly not...since that would make TOTAL sense.  I am for confusion, obsscurity, and spelling mistakes :smile: 

I'm going to leave that for a future PR, since there's another problem: `REQ+=1` gets missed in the for-loop somehow.  I'd like to understand why, and implement your suggestion as it will likely fix this too.